### ISSUE TITLE
what‘s the mean of ensemble?

### ISSUE THREAD

**Issue by HAOHAOXUEXIEN** [hzwer/Practical-RIFE#32]
if ensemble:
    f1, m1 = block[i](torch.cat((img1[:, :3], img0[:, :3], 1-timestep), 1), None, scale=scale_list[i])
    flow = (flow + torch.cat((f1[:, 2:4], f1[:, :2]), 1)) / 2
    mask = (mask + (-m1)) / 2

can explain the ensemble  effect? thanks very much
